capitaliq/src/test/scala/com/quantemplate/capitaliq/commands/IdentifierLoader.scala

```diff
@@ -0,0 +1,134 @@
+package com.quantemplate.capitaliq.commands
+
+import munit.FunSuite
+import org.mockito.Mockito.*
+import org.mockito.ArgumentMatchers.*
+import scala.concurrent.Future
+import scala.concurrent.ExecutionContext
+import akka.actor.typed.ActorSystem
+import java.nio.file.Paths
+import cats.syntax.option.*
+
+import com.quantemplate.capitaliq.qt.QTService
+import com.quantemplate.capitaliq.domain.CapitalIQ.Identifier
+import com.quantemplate.ActorSystemSuite
+import IdentifierLoader.*
+
+class IdentifierLoaderSpec extends ActorSystemSuite:
+  test("given a column name it should properly load the identifiers from QT dataset") {
+    given sys: ActorSystem[Nothing] = actorSystem()
```

Comment:
is actorSystem needed here for anything else than receiving ec? if not please not expose it as given (use plain val)

Reply:
It's used by `HttpService`, so it's not really used in here but I'd still want to expose it from the fixture for other tests.